mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -679,6 +679,12 @@ The `config` block supports:
   environment size are `ENVIRONMENT_SIZE_SMALL`, `ENVIRONMENT_SIZE_MEDIUM`,
   and `ENVIRONMENT_SIZE_LARGE`.
 
+* `resilience_mode` -
+  (Optional, Cloud Composer 2.1.15 or newer only)
+  The resilience mode states whether high resilience is enabled for 
+  the environment or not. Values for resilience mode are 
+  `RESILIENCE_MODE_UNSPECIFIED` and `HIGH_RESILIENCE`.
```

Comment:
If `RESILIENCE_MODE_UNSPECIFIED` is removed, we can remove it from the docs here too

Reply:
Done